presto-hive/src/main/java/com/facebook/presto/hive/rcfile/RcFilePageSourceFactory.java

```diff
@@ -109,12 +108,6 @@ public Optional<? extends ConnectorPageSource> createPageSource(
             HiveFileContext hiveFileContext,
             Optional<EncryptionInformation> encryptionInformation)
     {
-        if (!columns.isEmpty() && columns.stream().allMatch(hiveColumnHandle -> hiveColumnHandle.getColumnType() == AGGREGATED)) {
-            throw new UnsupportedOperationException("Partial aggregation pushdown only supported for ORC/Parquet files. " +
```

Comment:
curious. where does this check move after the refactoring. I wasn't able to find it. Is not needed any more.?

Reply:
tagged you where this check is moved to.  instead of adding error handling for every file format, we do it all in one place. that's why it's not needed here anymore.